webdriver/bidi/permission.json

```diff
@@ -3,7 +3,7 @@
     "bidi": {
       "permission": {
         "__compat": {
-          "spec_url": "https://w3c.github.io/webdriver-bidi/#module-permission",
+          "spec_url": "https://www.w3.org/TR/permissions/#webdriver-bidi-module-permissions",
```

Comment:
The permission module is special, and the WebDriver BiDi bridge didn't consider this yet. It's fixed there, but it'll still take a few days until we can merge all the latest BiDi support data (with parameters), so fixing here as well.